workloads/sample/sample.sh

```diff
@@ -20,11 +20,18 @@ run() {
 validate() {
     set -o pipefail
     export PYTHONPATH=${wt_build_dir}/lang/python:${wt_home_dir}/tools:$PYTHONPATH
+    failure_file_path=${failure_dir}/${failure_file}
+    database_path=$1/$database_dir
+    echo "Database path: $database_path"
+    echo "Database path: $failure_file_path"
```

Comment:
probably better to echo something different here? "Failure file path:"?

Reply:
Fixed in 526db78536b20ae757357623b3786e1a70c6084c